invenio_rdm_records/collections/services/service.py

```diff
@@ -131,3 +165,47 @@ def read_logo(self, identity, slug):
         if _exists:
             return url_for("static", filename=logo_path)
         raise LogoNotFoundError()
+
+    def read_many(self, identity, ids_, depth=2):
+        """Get many collections."""
+        self.require_permission(identity, "read")
+
+        if ids_ is None:
+            raise ValueError("IDs must be provided.")
+
+        if ids_ == []:
+            raise ValueError("Use read_all to get all collections.")
+
+        res = self.collection_cls.read_many(ids_, depth=depth)
+        return CollectionList(
+            identity, res, self.collection_schema, None, self.links_item_tpl
+        )
+
+    def read_all(self, identity, depth=2):
+        """Get all collections."""
+        self.require_permission(identity, "read")
+        res = self.collection_cls.read_all(depth=depth)
+        return CollectionList(
+            identity, res, self.collection_schema, None, self.links_item_tpl
+        )
+
+    def search_collection_records(self, identity, collection_or_id, params=None):
```

Comment:
```suggestion
    def search_records(self, identity, collection_or_id, params=None):
```

Reply:
That was the original name, I modified it after reviews :(